docs/howto/object-storage/s3/object-lock.md

```diff
@@ -204,7 +204,7 @@ use the following command:
 
     ```bash
     aws --profile <region> \
-      s3api put-legal-hold \
+      s3api put-object-legal-hold \
```

Comment:
Since the S3 API documentation says [legal holds can only be placed on individual object versions, not whole objects](https://docs.aws.amazon.com/AmazonS3/latest/userguide/batch-ops-legal-hold.html), do we want to add a `--version-id` option here?

Reply:
Indeed it is probably better to include `--version-id` since if one does not specify it, it will only apply it to the **last** version of that object. Fixed on commit [9866793](https://github.com/citynetwork/docs/pull/289/commits/9866793cceefa6b952740f844401d36c676aaec4).